Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json

```diff
@@ -0,0 +1,66 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "taken from https://github.com/tgstation/tgstation/raw/master/icons/turf/floors.dmi and edited by TheShuEd",
+  "size": {
```

Comment:
Needs the commit used.

Reply:
link copied from the original texture in ss14

Reply:
It's linking to tg's master which isn't valid, it should reference the commit it's pulled from please.

Reply:
problem: can't find this texture in tg build. It seems that they moved it somewhere very much, I've searched many files, but still can't find it.